admin/app/controllers/ContentPerformanceController.scala

```diff
@@ -194,4 +197,16 @@ object ContentPerformanceController extends Controller with AuthLogging with Log
         NoCache(Ok(views.html.contentLiveBlog("PROD", socialInteractionsChart, "Live Blog Performance", reportTimestamp)))
     }
   }
+
+  def renderVideoEncodingsDashboard() = AuthActions.AuthActionTest { request =>
+
+    val videoEncodingsReport = jobs.VideoEncodingsJob.getReport("missing-encodings")
+    videoEncodingsReport match {
+      case List() => NoCache(Ok("There are no reported encodings missing as of: %s".format(missingVideoEncodingDateTimeFormat.print(DateTime.now())) ))
```

Comment:
Also perfer

```scala
s"There are no reported encodings missing as of ${missingVideoEncodingDateTimeFormat.print(DateTime.now())}"
```

It does a macro expansion into a call that uses a StringBuilder, so it's more efficent than format. I also think it's more readable.